### ISSUE TITLE
Is there a systematic way to get the labels for the models

### ISSUE THREAD

**Issue by BoPeng** [osmr/pytorchcv#2]
The pre-trained models can be loaded, and can be applied to images to get probabilities, and ID of the categories. However, is there a systematic way to get the actual label for each classification?

**Comment by osmr** [comment]
Currently there is no such function in the pip-package.
